index.bs

```diff
@@ -9911,10 +9911,15 @@ Methods</h5>
 						{{AudioParamDescriptor/maxValue}}
 						in <var>descriptor</var>.
 
+					1. If <var>minValue</var> is
+						greater than or equal <var>maxValue</var>,
+						<span class="synchronous">throw an
+						{{InvalidStateError}}</span>.
```

Comment:
Per https://github.com/WebAudio/web-audio-api/pull/2174#issuecomment-595461280 this is unnecessary, but it's fine to keep it.

Reply:
Good point; I forgot about that. I think it's much clearer to remove this and adjust the next step to say that if the relationship minValue <= defaultValue <= maxValue is false, throw an error.